pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -138,6 +138,12 @@ Before running the following command, ensure that your `dkp` configuration **ref
     dkp upgrade kommander -v 4
     ```
 
+    If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress", you can trigger a reconciliation of the `HelmRelease` using the following command:
+
+    ```bash
+    kubectl annotate --overwrite helmrelease/<HELMRELEASE_NAME> -n <WORKSPACE_NAMESPACE> reconcile.fluxcd.io/requestedAt="$(date +%s)"
```

Comment:
i guess even this instruction needs to be switched to use `suspend` field?

Reply:
woops yeah missed this one thanks